bot/resources/tags/string-formatting.md

```diff
@@ -0,0 +1,27 @@
+**String Formatting Mini Language**
+The String Formatting Language in Python is a powerful way to tailor the display of strings and other data structures. This string formatting mini language works for f-strings and .format().
+
+Take a look at some of these examples!
+```py
+my_num = 2134234523
+print(f"{my_num:,}")
+
+my_smaller_num = -30.0532234
+print(f"{my_smaller_num:=09.2f}")
+
+my_str = "Center me!"
+print(f"{my_str:^20}")
+
+repr_str = "Spam \t Ham"
+print(f"{repr_str!r}")
```

Comment:
I feel like this is a bad example, it doesn't actually showcase that we're doing the same as `rep()`. Because it outputs exacltly the string you typed.

We should find something that has a different `__repr__` than `__str__`, or at least point out that the quotes were included when we printed it.

Reply:
Why doesn't it showcase this? The `\t` is for tabs and the repr is showcasing that the tab isn't rendered in the repr version.

Reply:
I feel like that can easily be mixed up because the string quite literally does `\t`.